core/tests/protractor_utils/LibraryPage.js

```diff
@@ -191,15 +191,12 @@ var LibraryPage = function() {
   };
 
   this.clickCreateActivity = async function() {
```

Comment:
If this PR is for fixing the editor page, why are there changes in the library page too? This isn't necessarily a problem, I'm just asking in case you didn't mean to include these changes

Reply:
I didn't make changes in libraraypage.js

Reply:
I see these commits in this PR:

![Screen Shot 2021-04-17 at 23 17 14](https://user-images.githubusercontent.com/19878639/115133043-1db13280-9fd3-11eb-92dc-919dec5ef3c6.png)